backend/src/main/java/com/bakdata/conquery/models/forms/util/DefaultSelectConceptManipulator.java

```diff
@@ -57,8 +57,8 @@ public void consume(CQConcept concept, Namespaces namespaces) {
 		switch(method) {
 			
 			case ADD_TO_COMPLETE_EMPTY:
-				Optional<Boolean> tablesEmpty = concept.getTables().stream().map(CQTable::getSelects).map(List::isEmpty).reduce(Boolean::logicalOr);
-				if(!(concept.getSelects().isEmpty() && tablesEmpty.orElse(true /* No table present -> signal empty tables*/))) {
+				Optional<Boolean> allTablesEmpty = concept.getTables().stream().map(CQTable::getSelects).map(List::isEmpty).reduce(Boolean::logicalAnd);
+				if(!(concept.getSelects().isEmpty() && allTablesEmpty.orElse(true /* No table present -> signal empty tables*/))) {
```

Comment:
das orElse(true) kannst du hochziehen und dann den Kommentar in die Zeile drüber machen.